gemrb/core/Interface.cpp

```diff
@@ -75,7 +75,7 @@
 #include "GUI/WorldMapControl.h"
 #include "RNG.h"
 #include "Scriptable/Container.h"
-#include "System/FileStream.h"
+#include "Streams/FileStream.h"
 #include "System/FileFilters.h"
 
```

Comment:
Below (line 85), 
```cpp
#include "CodepageToIconv.h"
```
does not work like this anymore

Reply:
right, we probably need to make it so that cmake includes the platform directory as an include directory.

Reply:
Easily done at the main CMakeLists.txt:478

Reply:
@MarcelHB did that do the trick?

Reply:
Looks good.